vagrant/Vagrantfile

```diff
@@ -339,7 +369,34 @@ Vagrant.configure('2') do |config|
                            mkdir -p /mnt/mdt2
                            mount -t lustre /dev/mapper/mpathd /mnt/mdt2
                          SHELL
+
+        mds.vm.provision 'create-ldiskfs-lvm-fs',
+                         type: 'shell',
+                         run: 'never',
+                         path: 'scripts/create_ldiskfs_lvm_fs.sh',
+                         args: ['fs', '/dev/mapper/mpathc', 1, '']
+
+        mds.vm.provision 'mount-ldiskfs-lvm-fs',
+                         type: 'shell',
+                         run: 'never',
+                         inline: <<-SHELL
+                           mkdir -p /mnt/mdt1
+                           mount -t lustre /dev/mapper/mdt1_vg-mdt /mnt/mdt1
+                         SHELL
+
       end
+
+      mds.vm.provision 'ha-ldiskfs-lvm-fs-step1',
```

Comment:
nit: something more descriptive that "step1"?

Reply:
something like:
ha-ldiskfs-lvm-fs-prep
ha-ldiskfs-lvm-fs-complete